torch/csrc/api/include/torch/nn/modules/loss.h

```diff
@@ -747,6 +747,45 @@ struct TORCH_API CrossEntropyLossImpl : public Cloneable<CrossEntropyLossImpl> {
 /// module storage semantics.
 TORCH_MODULE(CrossEntropyLoss);
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ CrossEntropyLossWithSoftLabels ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+/// Creates a criterion that combines :func:`nn.LogSoftmax` and
+/// :func:`nn.KLDivLoss` in one single class.
```

Comment:
nit: torch:nn::LogSoftmax and torch::nn::KLDivLoss (is that what they're called?)

Reply:
Note that this is consistent with the C++ module `CrossEntropyLoss` docs, so I'll make an update there as well.